### PR TITLE
[DRTV] Fix issue where sign language version of video would be downloaded

### PR DESCRIPTION
**Pull request by RasmusRendal**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Bug fix
- [ ] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Explanation of your *pull request* in arbitrary form goes here. Please make sure the description explains the purpose and effect of your *pull request* and is worded well enough to be understood. Provide as much context and examples as possible.
Fixing issue [13013](https://github.com/rg3/youtube-dl/issues/13013) where the sign language version of a video was downloaded, instead of the actual video.

### CODE REVIEW

**Comment by dstftw** [comment]
1. Do not skip formats.
2. Using single quotes.
3. Squash commits.

**Comment by RasmusRendal** [comment]
Should I just use the format flag for differing between the sign language version and the regular one? I was of the impression that the formats were for video quality.

**Comment by dstftw** [comment]
What you should do is to assign lower `preference` for sign language versions.

**Comment by RasmusRendal** [comment]
Alright, I did all the things now. Does this look good?

File: youtube_dl/extractor/drtv.py [hunk 1]
```diff
@@ -85,7 +97,11 @@ def _real_extract(self, url):
             kind = asset.get('Kind')
             if kind == 'Image':
                 thumbnail = asset.get('Uri')
-            elif kind in ('VideoResource', 'AudioResource'):
+            preference = 0
```
Comment (dstftw) [inline youtube_dl/extractor/drtv.py]: What's the point extracting it here?

File: youtube_dl/extractor/drtv.py [hunk 2]
```diff
@@ -85,7 +97,11 @@ def _real_extract(self, url):
...
+
+            sign_language = asset.get('Target') == 'SignLanguage'
```
Comment (dstftw) [inline youtube_dl/extractor/drtv.py]: What's the point extracting it as separate variable?

File: youtube_dl/extractor/drtv.py [hunk 3]
```diff
@@ -85,7 +97,11 @@ def _real_extract(self, url):
...
+
+            if kind in ('VideoResource', 'AudioResource'):
```
Comment (dstftw) [inline youtube_dl/extractor/drtv.py]: What's the point making it `if` when it should be `elif`?

File: youtube_dl/extractor/drtv.py (hunk 2 above)
Comment (RasmusRendal) [inline youtube_dl/extractor/drtv.py]: spoken_language worked like that, so I just assumed that was standard.

Comment (dstftw) [inline youtube_dl/extractor/drtv.py]: Ok, it's fine to separate it in order not to calculate it every time in a loop over links, but it should be located near `spoken_subtitles`.

**Pull request closed without merging**